src/rust/src/pkcs12.rs

```diff
@@ -388,7 +399,7 @@ enum CertificateOrPKCS12Certificate {
     PKCS12Certificate(pyo3::Py<PKCS12Certificate>),
 }
 
-fn serialize_bags<'p>(
+fn serialize_safebags<'p>(
```

Comment:
Looks like this slipped through the cracks for the refactor PR. I'm inclined to leave it in, but it really doesn't matter either way.

Reply:
Doesn't matter to me